doc/main.html

```diff
@@ -527,7 +527,10 @@ <h4>Normative references section</h4>
           <li>Each <code>li</code> element may contain one <code>a</code> element that contains a location where the reference can
           be retrieved.</li>
         </ul>
-
+          <p>Undated references should always be used (to designate using the latest version), except for when needed, as per <a href="#bib-iso-directives-part2"></a>. 
```

Comment:
```suggestion
          <p>Undated references should be used, since they point to the most most up-to-date edition of a document.</p>
          
          <p>Dated references may be used when referencing a specific element of the document is necessary, e.g., a table, or when past and future editions of the document are likely to be inadequate. </p>
```